go/bundle/cmd/sign-bundle/signedexchange.go

```diff
@@ -73,6 +75,10 @@ func addSignature(b *bundle.Bundle, signer *signature.Signer) error {
 }
 
 func SignExchanges(privKey crypto.PrivateKey) error {
+	if _, ok := privKey.(ecdsa.PrivateKey); !ok {
+		return errors.New("Private key is not ECDSA type.")
```

Comment:
Note: Actually ed25519 key is [allowed](https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#name-the-signature-header) in Signed Exchanges but we don't support it yet.